### DISCUSSION TITLE
Sabnzb script broken overnight

### DISCUSSION THREAD

**Discussion by aviduser2015** [tteck/Proxmox#2099] [Help Needed]
Hi there the SabNZB script has broken 

[ERROR] in line 23: exit code 4: while executing command wget -q http://http.us.debian.org/debian/pool/non-free/u/unrar-nonfree/$RELEASE

**Comment by mynted** [comment]
I'm also getting the same error:
[ERROR] in line 23: exit code 4: while executing command wget -q http://http.us.debian.org/debian/pool/non-free/u/unrar-nonfree/$RELEASE

**Reply by tteck** [reply]
fixed

**Reply by mynted** [reply]
Thank you so much!
